README.md

```diff
@@ -1,4 +1,16 @@
-# CascadeBot
+[![banner](https://user-images.githubusercontent.com/15330699/59548967-1f7cbb80-8f4e-11e9-9f98-af047ac30a2f.png)](https://cascadebot.org)
+
+<p align="center">
+    <a href="https://github.com/CascadeBot/CascadeBot/blob/dev/LICENSE"><img src="https://img.shields.io/github/license/cascadebot/cascadebot.svg?style=flat-square" alt="GitHub License"></a>
+    <a href="https://jenkins.cascadebot.org/job/Cascade%20Bot/lastBuild/"><img src="https://img.shields.io/jenkins/build/https/jenkins.cascadebot.org/cascade%20bot.svg?style=flat-square" alt="Jenkins"></a>
+    <a href="https://github.com/CascadeBot/CascadeBot/issues"><img src="https://img.shields.io/github/issues/cascadebot/cascadebot.svg?color=purple&style=flat-square" alt="GitHub Issues"></a>
+    <a href="https://github.com/CascadeBot/CascadeBot/pulls"><img src="https://img.shields.io/github/issues-pr/cascadebot/cascadebot.svg?color=purple&style=flat-square" alt="GitHub Pull Requests"></a>
+    <a href="https://github.com/CascadeBot/CascadeBot/network/members"><img src="https://img.shields.io/github/forks/cascadebot/cascadebot.svg?style=flat-square" alt="GitHub Forks"></a>
+    <a href="https://github.com/CascadeBot/CascadeBot/stargazers"><img src="https://img.shields.io/github/stars/cascadebot/cascadebot.svg?style=flat-square" alt="GitHub Stars"></a>
+    <a href="https://cascadebot.org"><img src="https://img.shields.io/website/https/cascadebot.org.svg?down_color=red&down_message=offline&style=flat-square&up_message=online" alt="Website"></a>
+    <br><br>
+    <a href="https://discord.gg/P23GZFB"><img src="https://discordapp.com/api/guilds/488394590458478602/widget.png?style=banner2" alt="Discord Server"></a>
+</p>
 
 CascadeBot runs on JDK 11 to have the latest features available to provide the best user experience.
 
```

Comment:
We're really going to need more information here. Even if it's just a list of developers or the plugin's main features, they should be listed both on the website and here.

Reply:
Oh I didn't see the PR comment, I'm retarded don't worry

Reply:
👏

Reply:
Good so far :p